components/bl-static-breadcrumbs/src/index.js

```diff
@@ -0,0 +1,26 @@
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function StaticBreadcrumbsComponent({ component, eventHandlers, elRef }) {
+  const { classList, display, style, options } = component;
+  const { onItemClick, onMouseOver, onMouseOut } = eventHandlers;
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div ref={ elRef } className={ cn('bl-customComponent-staticBreadcrumbs', classList) } style={ style }>
+      { options?.map(item => (
+        <div className="breadcrumbs-item">
+          <span
+            className="breadcrumbs-item-label"
+            onClick={ () => onItemClick({ item }) }
+            onMouseOver={ () => onMouseOver({ item }) }
+            onMouseOut={ () => onMouseOut({ item }) }>
```

Comment:
How about to make two context blocks? I think it will be more prettier if user have `Page Label` and `Page Name` context blocks.